<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="HomeTenant-20240508181659" tests="1" file=".py" time="160.401" timestamp="2024-05-08T18:19:40" failures="0" errors="1" skipped="0">
	<testcase classname="HomeTenant" name="test_start_tenant" time="160.401" timestamp="2024-05-08T18:19:40" file="Home.py" line="19">
		<error type="NoSuchWindowException" message="Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=124.0.6367.119)
Stacktrace:
	GetHandleVerifier [0x00007FF6200D1572+60802]
	(No symbol) [0x00007FF62004AC72]
	(No symbol) [0x00007FF61FF07CE4]
	(No symbol) [0x00007FF61FEDDFDF]
	(No symbol) [0x00007FF61FF81E57]
	(No symbol) [0x00007FF61FF998D1]
	(No symbol) [0x00007FF61FF7A923]
	(No symbol) [0x00007FF61FF48FEC]
	(No symbol) [0x00007FF61FF49C21]
	GetHandleVerifier [0x00007FF6203D420D+3217949]
	GetHandleVerifier [0x00007FF6204161A7+3488183]
	GetHandleVerifier [0x00007FF62040F12F+3459391]
	GetHandleVerifier [0x00007FF62018B936+823622]
	(No symbol) [0x00007FF62005600F]
	(No symbol) [0x00007FF620050F34]
	(No symbol) [0x00007FF6200510C2]
	(No symbol) [0x00007FF620041914]
	BaseThreadInitThunk [0x00007FFEF342257D+29]
	RtlUserThreadStart [0x00007FFEF4B0AA48+40]
"><![CDATA[Traceback (most recent call last):
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\Home.py", line 54, in test_start_tenant
    validate_text(self.driver, titlle_value1, value_expected1)
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\Elements.py", line 456, in validate_text
    elemento = WebDriverWait(driver, 10).until(
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\support\wait.py", line 96, in until
    value = method(self._driver)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\support\expected_conditions.py", line 84, in _predicate
    return driver.find_element(*locator)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 742, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 348, in execute
    self.error_handler.check_response(response)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=124.0.6367.119)
Stacktrace:
	GetHandleVerifier [0x00007FF6200D1572+60802]
	(No symbol) [0x00007FF62004AC72]
	(No symbol) [0x00007FF61FF07CE4]
	(No symbol) [0x00007FF61FEDDFDF]
	(No symbol) [0x00007FF61FF81E57]
	(No symbol) [0x00007FF61FF998D1]
	(No symbol) [0x00007FF61FF7A923]
	(No symbol) [0x00007FF61FF48FEC]
	(No symbol) [0x00007FF61FF49C21]
	GetHandleVerifier [0x00007FF6203D420D+3217949]
	GetHandleVerifier [0x00007FF6204161A7+3488183]
	GetHandleVerifier [0x00007FF62040F12F+3459391]
	GetHandleVerifier [0x00007FF62018B936+823622]
	(No symbol) [0x00007FF62005600F]
	(No symbol) [0x00007FF620050F34]
	(No symbol) [0x00007FF6200510C2]
	(No symbol) [0x00007FF620041914]
	BaseThreadInitThunk [0x00007FFEF342257D+29]
	RtlUserThreadStart [0x00007FFEF4B0AA48+40]

]]></error>
		<system-out><![CDATA[Tiempo de espera agotado. El texto por xpaht no estÃ¡ presente.
]]></system-out>
	</testcase>
</testsuite>
